Adafruit_FreeTouch.cpp

```diff
@@ -27,6 +27,10 @@
 
 #include "adafruit_ptc.h"
 
+#if !defined(__SAMD21G18A__) and !defined(__SAMD11C14A__)
```

Comment:
We support all SAMD21 variants. Please update so they'll all work. I like the error!